### PR TITLE
Add pojo router tests

### PR DESCRIPTION
**Pull request by melissafzhang**
* This PR adds tests for the pojo router
* Moves dependencies like lerna and types to the appropriate dev dependencies
* Removes `useBestMatch` and bumps version to 0.2.0

### CODE REVIEW

File: packages/pojo-router/package.json [hunk 1]
```diff
@@ -53,8 +57,6 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.10.5",
-    "@types/lodash": "^4.14.158",
-    "lerna": "^3.22.1",
```
Comment (melissafzhang) [inline packages/pojo-router/package.json:57]: Moved this to dev dependencies which fixes our salus as well

File: packages/pojo-router/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 0.2.0 (2020-08-04)
```
Comment (ntucker) [inline packages/pojo-router/CHANGELOG.md:6]: version bumps and changelogs happen autmatically when runnign 'yarn release' no need to include them here.

File: salus.yaml [hunk 3]
```diff
@@ -2,10 +2,6 @@ scanner_configs:
   YarnAudit:
     exclude_groups:
     - devDependencies         # dev only dependencies
-    exceptions:
```
Comment (ntucker) [inline salus.yaml:5]: nice

File: packages/pojo-router/src/index.tsx [hunk 4]
```diff
@@ -33,7 +33,7 @@ type NamedPath = {
   [k: string]: any;
 };
 type Route = readonly [string, AnyIfEmpty<DefaultRoutePojo>];
-type Props = {
+export type Props = {
```
Comment (ntucker) [inline packages/pojo-router/src/index.tsx:36]: since we're exporting maybe give a longer name?

**Pull request merged**